files/zh-cn/web/api/blob/slice/index.md

```diff
@@ -1,32 +1,34 @@
 ---
-title: Blob.slice
+title: Blob: slice() 方法
+short-title: slice()
```

Comment:
Please remove the [short-title](https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md#%E5%85%83%E6%95%B0%E6%8D%AE)

```suggestion
title: Blob：slice() 方法
```

Reply:
ok